app/models/inning.rb

```diff
@@ -49,4 +59,10 @@ def create_turing_modules
       turing_modules.create!(program: 'BE', module_number: i + 1)
     end
   end
+
+  def date_validation(current_inning_date)
+    return errors.add(:start_date, "Can't be blank") if start_date.nil?
```

Comment:
This would be a good place to use `.blank?` instead of `.nil` since blank will also catch an empty string.

Reply:
awesome TY~! Updated to use .blank?